service/handlers.go

```diff
@@ -669,6 +709,16 @@ func (h handler) getProject(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	level.Debug(l).Log("message", "getting project from ddb", "db-type", "dynamo")
+	if _, err := h.ddbClient.ReadProjectEntry(ctx, projectName); err != nil {
+		if errors.Is(err, db.ErrProjectNotFound) {
+			level.Warn(l).Log("message", "project does not exist in ddb", "db-type", "dynamo")
+		} else {
+			level.Warn(l).Log("message", "error retrieving project from ddb", "db-type", "dynamo", "error", err)
```

Comment:
does it make sense to also log the project name?

Reply:
it's already set in the logger [here](https://github.com/thbishop-intuit/cello/blob/bacd4b3457609d9bcaad5db909dd15e6f25fcd37/service/handlers.go#L685).